assets/express/src/api/user/index.ts

```diff
@@ -0,0 +1,14 @@
+import { ModuleFactory } from '@common';
+import { userServiceFactory } from '@services';
+import { UserController } from './interfaces';
+import { userControllerFactory } from './v1';
```

Comment:
Here we can just have `import * as version1Controllers from './v1';` Otherwise for each version we would have
```
import { userControllerFactoryV1 } from './v1';
import { userControllerFactoryV2 } from './v2';
```